### PR TITLE
Dev/php7

### PR DESCRIPTION
**Pull request by ezzy1337**

### CODE REVIEW

File: src/Queue.php [hunk 1]
```diff
@@ -439,10 +439,13 @@ private function ensureIndex(array $index)
                         return;
                     }
                 }
+
+                //@codeCoverageIgnoreStart
```
Comment (chrisryan) [inline src/Queue.php]: :confused: What was I thinking! I would vote to have this removed.

Comment (ezzy1337) [inline src/Queue.php]: Your wish is my command.

File: .travis.yml [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+  - EXT_VERSION=1.1 DB_PACKAGE=mongodb-org
 before_install:
   - "sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10"
   - "echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list"
   - "sudo apt-get update"
   - "sudo apt-get install ${DB_PACKAGE}"
 before_script:
   - composer self-update
-  - yes '' | pecl install -f mongo-${EXT_VERSION}
+  - yes '' | pecl install -f mongodb-${EXT_VERSION}
```
Comment (chadicus) [inline .travis.yml]: Since you goal is to just make the library php7 compatible, I would not worry about upgrading the mongo ext.

Comment (chrisryan) [inline .travis.yml]: Php7 does not support the older mongo-ext library.

**Pull request merged**